oak_functions/loader/src/server.rs

```diff
@@ -816,13 +831,25 @@ fn oak_functions_resolve_func(field_name: &str) -> Option<(usize, wasmi::Signatu
 /// `wasmi` specific result type `Result<Option<wasmi::RuntimeValue>, wasmi::Trap>`, mapping:
 /// - `Ok(())` to `Ok(Some(OakStatus::Ok))`
 /// - `Err(x)` to `Ok(Some(x))`
-fn map_host_errors(
+fn from_oak_status_result(
     result: Result<(), OakStatus>,
 ) -> Result<Option<wasmi::RuntimeValue>, wasmi::Trap> {
-    Ok(Some(wasmi::RuntimeValue::I32(result.map_or_else(
-        |x: OakStatus| x as i32,
-        |_| OakStatus::Ok as i32,
-    ))))
+    let oak_status_from_result = result.map_or_else(|x: OakStatus| x, |()| OakStatus::Ok);
+    let wasmi_value = wasmi::RuntimeValue::I32(oak_status_from_result as i32);
+    Ok(Some(wasmi_value))
+}
+
+/// A helper function to move between our specific result type `Result<(), ChannelStatus>` and the
+/// `wasmi` specific result type `Result<Option<wasmi::RuntimeValue>, wasmi::Trap>`, mapping:
+/// - `Ok(())` to `Ok(Some(ChannelStatus::Ok))`
+/// - `Err(x)` to `Ok(Some(x))`
+fn from_channel_status_result(
```

Comment:
@rbehjati I renamed both to `from_channel_status_result` and `from_oak_status_result` and refactored a bit. WDYT?

Reply:
I like these names. Thanks :)